src/apps/backend/modules/account/internal/store/account-db.ts

```diff
@@ -13,6 +15,13 @@ export const AccountDbSchema: Schema = new Schema<AccountDB>(
       type: Boolean,
       required: true,
     },
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
```

Comment:
After I gave you the feedback on keeping it optional, I thought a bit more about it. I think we want to make last name as required as well.

Reply:
its minimum length will be 1 and error message will be `Last name should not be empty` is it fine sir?